Dockerfile

```diff
@@ -1,6 +1,6 @@
 # Set default values for build arguments
-ARG DEFRA_VERSION=2.4.0
-ARG BASE_VERSION=22.9.0-alpine3.20
+ARG DEFRA_VERSION=2.4.1
```

Comment:
Not sure if we have a properly defined process for bumping the Defra version, but as this is a "biggish" change maybe we should go to 2.5.0?

Reply:
I think you're being generous with this comment as the README clearly states I've got this wrong!

I looked through the history and saw it was inconsistent, so just opted for patch.  However, the README says that updating Node or Alpine should be a minor update.

I have corrected it. :)